tools/c7n_azure/c7n_azure/resources/sqlserver.py

```diff
@@ -23,3 +25,20 @@ class resource_type(ArmResourceManager.resource_type):
         service = 'azure.mgmt.sql'
         client = 'SqlManagementClient'
         enum_spec = ('servers', 'list', None)
+
+
+@SqlServer.filter_registry.register('sql-database-view')
+class SqlDatabaseViewFilter(Filter):
+
+    schema = type_schema('sql-database-view')
+
+    def __call__(self, i, *args, **kwargs):
+
+        if 'databases' not in i:
+            client = self.manager.get_client()
+            resource_group = i['resourceGroup']
+            server_name = i['name']
+            dbs = client.databases.list_by_server(resource_group, server_name)
+            i['databases'] = [db.as_dict() for db in dbs]
```

Comment:
we always prefix annotations ue, `c7n:databases` on to resources to disambiguate against any intrinsic attributes.